test_howdoi.py

```diff
@@ -26,6 +28,9 @@ def setUp(self):
         self.bad_queries = ['moe',
                             'mel']
 
+    def assertValidResponse(self, res):
+        return len(res) > 0
+
```

Comment:
Not sure if this is related to the issue at hand. Also, what was wrong with the previous way of testing?

Reply:
We noticed that when we added the JSON response, all the tests started failing.

Reply:
But let's make it `self.assertTrue(len(res) > 0)` so we can use the unitest's version of assert